src/CommunityToolkit.Maui.BindablePropertySG/Helpers/EquatableArray{T}.cs

```diff
@@ -0,0 +1,200 @@
+﻿//https://github.com/CommunityToolkit/dotnet/blob/main/src/CommunityToolkit.Mvvm.SourceGenerators/Helpers/EquatableArray%7BT%7D.cs
```

Comment:
We need this because we will work with a collection of BindableProperties... And all the other collections didn't work because they aren't very good with `Equals`, so I grabbed this one from Sergio, and works like a charm.